test/integration_test/lib/util/test_author.py

```diff
@@ -0,0 +1,11 @@
+from lib.util.author import *
+
+def test_get_uid_map():
+
+	author_uid_map=dict()
+	check='jdelvare@suse.de' in author_uid_map
```

Comment:
These checks before the test are not necessary.

Reply:
Try to write simple and short test cases. This whole test can be summarised to `assert 'jdelvare@suse.de' in get_uid_map()`. It makes it easy to read and edit them.